### PR TITLE
Adds Player-Crafted Jewellery… Made from Bullets: Necklaces

### PR DESCRIPTION
**Pull request by DoctorBoomstick**
#### Summary
Content " adds the capability for players to craft simple and more advanced necklaces and pendants from spent bullet casings."
#### Purpose of change
I was originally inspired to put up this PR after seeing somebody on Discord joking about making apparel with bullets, and I thought to myself, Hey, that sounds like a pretty good idea for adding some roleplay opportunities. Some searching later and discovering that bullet jewellery is actually a thing that people sometimes make, I supposed that, while being a craft that doesn’t add anything tangible to game play, it would be a nice addition to allow roleplay while also expanding the fleet of post-Cataclysm aesthetic items, because you know that somebody will inevitably try to make something pretty out of the screaming flesh apocalypse.
At the end of the day, I just personally like the idea of using a marksman’s rifle to snipe a high-priority enemy and then wearing the killing bullet's casing as a trophy. Also, the gem-setting proficiency really needed a use case, and I’m a dumb boomstick man, not a refined fashion man.
#### Describe the solution
In today’s episode of Boomstick adds something that doesn’t need to exist, I present:
- The addition of three sorts of basic cartridge necklaces: pistol, rifle, and heavy ordinance.
- The addition of three advanced forms of necklaces using pre-existing pendent chains and gems: pistol, rifle, and heavy ordinance.
- The addition of respective crafting recipes for all six new items.
#### Describe alternatives you've considered
I considered:
- Opening the throttle wide and also adding bullet earrings, bracelets, and rings. I abstained in case this style of item is considered so useless that it doesn’t merit inclusion in the game and because I’d require the additional addition of earing backings for the cartridge earings, an item that I have no understanding of and no will to touch item spawn groups and disassemble recipes to add them. If this PR is accepted, I’ll be adding other bullet crafts in follow-up PRs.
- Adding spawns for the advanced bullet necklaces to jewellery stores. I left out adding anything to spawn groups and leaving them as strictly player-crafted goods, as, while a thing that people sell, I feel like their quite niche to represent in the commercial market.
#### Testing
To be done, but I tested it back when I added the first form of necklace, bug fixed, and just copied and pasted the same format for the rest of them. This being said, I expect the tests to detonate because GitHub PR tests hate me.
#### Additional context
A link to a list of DIY guides that I found discussing the making of bullet jewellery.
https://somethingborrowedpdx.com/diy-bullet-jewelry/
A link to a website that commercially makes and sells cartridge jewellery.
https://www.jectz.com/

### CODE REVIEW

**Comment by Karol1223** [comment]
``Jewellery`` is British English, while ``Jewelry`` is American English, according to Google (and I assume that's why the spellchecker is freaking out). Could you change that?

**Comment by BlizzDA** [comment]
>A simple, if rather `crewed`, piece of post-Cataclysm 

While it doesn't tip the spellchecker the word should be `crude` to denote an unrefined state.

File: data/json/requirements/ammo.json [hunk 1]
```diff
@@ -309,5 +309,84 @@
         [ "reloaded_50beowulf_penetrator", -1 ]
       ]
     ]
+  },
+  {
+    "id": "pistol_casings",
+    "type": "requirement",
+    "//": "A list of all pistol-calibre casings for use in crafts and adjacent projects.",
```
Comment (TheShadowFerret) [inline data/json/requirements/ammo.json:316]:
```suggestion
    "//": "A list of all pistol-caliber casings for use in crafts and adjacent projects.",
```

File: data/json/requirements/ammo.json [hunk 2]
```diff
@@ -309,5 +309,84 @@
...
+        [ "9x18mm_casing", 1 ],
+        [ "380_casing", 1 ],
+        [ "50ae_casing", 1 ]
+      ]
+    ]
+  },
+  {
+    "id": "rifle_casings",
+    "type": "requirement",
+    "//": "A list of rifle-calibre casings for use in crafts and adjacent projects. Excludes giant cartridges like .50 Browning and shotgun shells.",
```
Comment (TheShadowFerret) [inline data/json/requirements/ammo.json:346]:
```suggestion
    "//": "A list of rifle-caliber casings for use in crafts and adjacent projects. Excludes giant cartridges like .50 Browning and shotgun shells.",
```

**Review by TheShadowFerret — COMMENTED** [review]
~~Who dares bring British spellings to my American Cataclysm~~

**Comment by Kantonine** [comment]
another one is being reported for the item descriptions

**Pull request merged**